src/components/curation/GeneInformation.vue

```diff
@@ -1,9 +1,20 @@
 <script>
 export default {
+  data() {
+    return {
+      readMoreActivated: false,
+      MAX_CHARACTERS: 800
```

Comment:
better to keep this constant in Constants.js file and import it here

Reply:
You mean only the MAX_CHARACTERS?

Reply:
yes